orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/ServerGroupCacheForceRefreshTask.groovy

```diff
@@ -73,6 +73,11 @@ class ServerGroupCacheForceRefreshTask extends AbstractCloudProviderAwareTask im
     }
 
     boolean allAreComplete = processPendingForceCacheUpdates(account, cloudProvider, stageData, stage.startTime)
+    if (allAreComplete) {
+      // ensure clean stage data such that a subsequent ServerGroupCacheForceRefresh (in this stage) starts fresh
+      stageData.reset()
```

Comment:
We have a couple stages that have multiple `ServerGroupCacheForceRefreshTask`'s. 

This is an attempt at cleaning up the context such that subsequent tasks will start fresh.